GOVERNANCE_pme.md

```diff
@@ -0,0 +1,188 @@
+
+## Community Roles and Path to Maintainer
+
+Roles apply equally to code and non-code activities (e.g., documentation, community, tools, streams). The following table lists the roles within the PyPSA-meets-Earth community, describing:
+
+- General responsibilities expected by individuals in each role
+- Requirements necessary to join or remain in a role
+- Associated access rights
+
+<table>
+  <thead>
+    <tr>
+      <th>Role</th>
+      <th>Responsibilities</th>
+      <th>Requirements</th>
+      <th>Access Rights</th>
+      <th>Scope</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Participant</td>
+      <td>
+        <ul>
+          <li>Follow the <a href="https://lfprojects.org/policies/code-of-conduct/">Code of Conduct</a></li>
+          <li>Engage in discussions, submit issues and PRs, contribute to code or non-code streams</li>
+        </ul>
+      </td>
+      <td>None</td>
+      <td>
+        <ul>
+          <li>Can submit PRs and issues from forks</li>
+          <li>Can join the PyPSA-meets-Earth Discord channel (<a href="https://discord.gg/AnuJBk23FU">join here</a>)</li>
+          <li>Can take part in community discussions</li>
+        </ul>
+      </td>
+      <td>GitHub organization</td>
+    </tr>
+    <tr><td colspan="5"><em>Inherits from Participant</em></td></tr>
+    <tr>
+      <td>Contributor</td>
+      <td>
+        <ul>
+          <li>Report and help resolve issues</li>
+          <li>Answer community questions</li>
+          <li>Review and provide feedback on issues/PRs</li>
+          <li>Contribute to code, documentation, tools, or community streams</li>
+        </ul>
+      </td>
+      <td>
+        <ul>
+          <li>Active participation (e.g., merged contribution, regular discussion, or content creation)</li>
+        </ul>
+      </td>
+      <td>
+        <ul>
+          <li>Can review PRs</li>
+          <li>May close/open/reassign issues</li>
+          <li>May request PR reviews</li>
+          <li>Can mark duplicates</li>
+        </ul>
+      </td>
+      <td>Specific repo(s) or streams under pypsa-meets-earth</td>
+    </tr>
+    <tr><td colspan="5"><em>Inherits from Contributor</em></td></tr>
+    <tr>
+      <td>Maintainer</td>
+      <td>
+        <ul>
+          <li>Guide project direction and quality</li>
+          <li>Help cut releases</li>
+          <li>Mentor contributors</li>
+          <li>Review and merge PRs</li>
+          <li>Coordinate across streams (code, docs, tools, community)</li>
+        </ul>
+      </td>
+      <td>
+        <ul>
+          <li>Consistent, high-quality contributions</li>
+          <li>Broad project knowledge</li>
+        </ul>
+      </td>
+      <td>
+        <ul>
+          <li>Approve and merge PRs</li>
+          <li>Manage releases</li>
+          <li>Moderate discussions</li>
+          <li>Vote on major decisions (if needed)</li>
```

Comment:
Would be good to discuss what are the `Access Rights` for non-code streams.

Reply:
To clarify: I like the idea of "Access Rights to fulfill the goal of the stream/package", but don't we need a distinction between roles (e.g. Participant, Contributor, etc.)?